### PR TITLE
OffroadHome: merge stylesheets

### PR DESCRIPTION
**Pull request by deanlee**

### CODE REVIEW

File: selfdrive/ui/qt/home.cc [hunk 1]
```diff
@@ -139,11 +137,23 @@ OffroadHome::OffroadHome(QWidget* parent) : QFrame(parent) {
...
   setStyleSheet(R"(
+    * {
+     color: white;
+    }
     OffroadHome {
       background-color: black;
     }
-    * {
-     color: white;
+    OffroadHome>QPushButton {
```
Comment (adeebshihadeh) [inline selfdrive/ui/qt/home.cc:146]: this is really only for the update button, the selector should be more specific

File: selfdrive/ui/qt/home.cc [hunk 2]
```diff
@@ -138,11 +137,23 @@ OffroadHome::OffroadHome(QWidget* parent) : QFrame(parent) {
...
   setStyleSheet(R"(
+    * {
+     color: white;
+    }
     OffroadHome {
       background-color: black;
     }
-    * {
-     color: white;
+    QPushButton#alert_notification {
```
Comment (adeebshihadeh) [inline selfdrive/ui/qt/home.cc:146]:
```suggestion
    #alert_notification {
```

**Comment by adeebshihadeh** [comment]
Want to fix #21266 too?

**Comment by deanlee** [comment]
Ok, let me check

**Comment by adeebshihadeh** [comment]
Hmm, think it's also fine to separate the update and alert buttons.

**Comment by deanlee** [comment]
I'll fix it in another PR

**Pull request merged**